app/src/main/java/com/github/polybooks/ListSalesActivity.kt

```diff
@@ -29,6 +28,10 @@ class ListSalesActivity : AppCompatActivity() {
     private val mLayout: RecyclerView.LayoutManager = LinearLayoutManager(this)
     private val initialBooks: List<Sale> = emptyList()
 
+    private val firestore = FirebaseFirestore.getInstance()
+    private val bookDB = OLBookDatabase { string -> url2json(string) }
+    private val salesDB = SaleDatabase(firestore, bookDB)
```

Comment:
It doesn't need to be done in this PR, but I feel like we may be ready now to have global Book and Sale databases somewhere in our app :)